front/app/containers/HomePage/index.tsx

```diff
@@ -40,8 +37,6 @@ const HomePage = () => {
     }
   }, [pressedLetterAKey, userHasAdminAccess]);
 
-  if (!homepageLayout) return <Spinner />;
-
```

Comment:
@IvaKop I feel like removing this shouldn't cause any errors, since down the chain we have this check as well when the homepageLayout is used:


 
```

  const getEditorData = () => {
    if (
      homepageLayout &&
      !isEmpty(homepageLayout.data.attributes.craftjs_json)
    ) {
      return homepageLayout.data.attributes.craftjs_json;
    } else {
      return undefined;
    }
  };
```

Reply:
I'll do a bit more functional testing, but I think it should be ok to merge this :thinking: